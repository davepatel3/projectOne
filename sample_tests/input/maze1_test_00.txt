3
3
3
0
print_walls
0
print_walls
1
print_walls
2
quit
