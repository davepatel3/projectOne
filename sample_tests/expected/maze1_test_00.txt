+---+---+---+
| X | X   X |
+   +---+   +
| X | F   X |
+   +---+---+
| F | F   F |
+---+---+---+

+---+---+---+
| F | F   X |
+---+---+   +
| F   B | F |
+---+   +---+
|   | B |   |
+---+---+---+

+---+---+---+
| B | B | X |
+   +   +   +
| B | X | B |
+---+   +   +
| B   X | B |
+---+---+---+

